diff --git a/sites/all/modules/contrib/rabbit_hole/modules/rh_node/rh-node.js b/sites/all/modules/contrib/rabbit_hole/modules/rh_node/rh-node.js
index 36dceea..62a9d20 100644
--- a/sites/all/modules/contrib/rabbit_hole/modules/rh_node/rh-node.js
+++ b/sites/all/modules/contrib/rabbit_hole/modules/rh_node/rh-node.js
@@ -3,6 +3,12 @@
 Drupal.behaviors.rhNode = {
   attach: function (context, settings) {
 
+    // The drupalSetSummary method required for this behavior is not available
+    // on some pages, so we need to make sure this
+    // behavior is processed only if drupalSetSummary is defined.
+    if (typeof jQuery.fn.drupalSetSummary == 'undefined') {
+      return;
+    }
     // Set the summary for the settings form.
     $('fieldset.rabbit-hole-settings-form').drupalSetSummary(function() {
       var $rabbitHoleAction = $('.rabbit-hole-action-setting input:checked');
